src/core/src/core_logic/PatchInstaller.py

```diff
@@ -306,7 +310,7 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
 
             # maintenance window check
             remaining_time = maintenance_window.get_remaining_time_in_minutes()
-            if maintenance_window.is_package_install_time_available(package_manager, remaining_time, number_of_packages_in_batch=1) is False:
+            if maintenance_window.is_package_install_time_available(package_manager, remaining_time, number_of_packages_in_batch=1) is False and self.is_install_patches_in_single_batch_enabled() == False:
```

Comment:
If single batch is enabled, we will not stop extension due to maintenance window timeoff.

Is there an agreement on this with the customer? Because CRP will still timeout after it's own interval and not return the completed status. i.e. customers will see patch in progress or errored (Portal/AUM, etc) when it actually succeeded in the machine